packages/accordion/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 ## Versions
 
+* [v7.0.8 - Removed developer dependecies of the packages](#v708)
```

Comment:
this implies that developer dependencies were removed for each of the packages. 

Try something like

"Remove `--save-dev` flag from read me instructions"